app/assets/stylesheets/preact/onboarding-modal.scss

```diff
@@ -1,438 +1,528 @@
 @import 'variables';
 @import 'mixins';
+@import 'components/buttons';
+@import 'components/modals';
+@import 'components/forms';
+@import '../vars/variables';
```

Comment:
Hmm.. It's weird but I can't check your PR locally because of broken paths above... Looks like you already may need to import the other file instead...